tools/tune/eq/eq_compute.m

```diff
@@ -41,7 +41,7 @@
 
 %% Define target (e.g. speaker) response as parametric filter. This could also
 %  be numerical data interpolated to the grid.
-if length(eq.parametric_target_response) > 0
+if ~isempty(eq.parametric_target_response)
```

Comment:
Are these length() vs. ~isempty() cleanup or Matlab compatibility?

Reply:
The execution speed of `isempty` is faster then comparing `length` to `0`, therefore the proposal is replace `length` with `isempty`